clash-prelude/CHANGELOG.md

```diff
@@ -1,453 +0,0 @@
-  * [#575](https://github.com/clash-lang/clash-compiler/issues/575)
-  * [#581](https://github.com/clash-lang/clash-compiler/issues/581)
-  * [#582](https://github.com/clash-lang/clash-compiler/issues/582)
-  * [#586](https://github.com/clash-lang/clash-compiler/issues/586)
-  * [#588](https://github.com/clash-lang/clash-compiler/issues/588)
-  * [#591](https://github.com/clash-lang/clash-compiler/issues/591)
-  * [#596](https://github.com/clash-lang/clash-compiler/issues/596)
-  * [#601](https://github.com/clash-lang/clash-compiler/issues/601)
-  * [#607](https://github.com/clash-lang/clash-compiler/issues/607)
-  * [#629](https://github.com/clash-lang/clash-compiler/issues/629)
-  * [#637](https://github.com/clash-lang/clash-compiler/issues/637)
-  * [#644](https://github.com/clash-lang/clash-compiler/issues/644)
-  * [#647](https://github.com/clash-lang/clash-compiler/issues/647)
-  * [#661](https://github.com/clash-lang/clash-compiler/issues/661)
-  * [#668](https://github.com/clash-lang/clash-compiler/issues/668)
-  * [#677](https://github.com/clash-lang/clash-compiler/issues/677)
-  * [#678](https://github.com/clash-lang/clash-compiler/issues/678)
-  * [#682](https://github.com/clash-lang/clash-compiler/issues/682)
-  * [#691](https://github.com/clash-lang/clash-compiler/issues/691)
-  * [#703](https://github.com/clash-lang/clash-compiler/issues/703)
-  * [#713](https://github.com/clash-lang/clash-compiler/issues/713)
-  * [#715](https://github.com/clash-lang/clash-compiler/issues/715)
-  * [#727](https://github.com/clash-lang/clash-compiler/issues/727)
-  * [#730](https://github.com/clash-lang/clash-compiler/issues/730)
-  * [#736](https://github.com/clash-lang/clash-compiler/issues/736)
-  * [#738](https://github.com/clash-lang/clash-compiler/issues/738)
-
-## 0.99.3 *July 26th 2018*
-* Bundle and BitPack instances up to and including 62-tuples
-* Handle undefined writes to RAM properly
-* Handle undefined clock enables properly
-
-## 0.99.1 *May 12th 2018*
-* Support for `ghc-typelits-natnormalise-0.6.1`
-* `Lift` instances for `TopEntity` and `PortName`
-* `InlinePrimitive` support: specify HDL primitives inline with Haskell code
-
-## 0.99 *March 31st 2018*
-* Major API overhaul: check the migration guide at the end of `Clash.Tutorial`
-* New features:
-  * Explicit clock and reset arguments
-  * Rename `CLaSH` to `Clash`
-  * Implicit/`Hidden` clock and reset arguments using a combination of
-    `reflection` and `ImplicitParams`.
-  * Large overhaul of `TopEntity`  annotations
-  * PLL and other clock sources can now be instantiated using regular functions:
-    `Clash.Intel.ClockGen` and `Clash.Xilinx.ClockGen`.
-  * DDR registers:
-    * Generic/ASIC: `Clash.Explicit.DDR`
-    * Intel: `Clash.Intel.DDR`
-    * Xilinx: `Clash.Intel.Xilinx`
-  * `Bit` is now a `newtype` instead of a `type` synonym and will be mapped to
-    a HDL scalar instead of an array of one (e.g `std_logic` instead of
-    `std_logic_vector(0 downto 0)`)
```

Comment:
Do we want to keep these?

Reply:
It seemed "recent" enough that people might be interested in reading it.

Reply:
Oh, I see what you're saying now. Yeah I guess.

Reply:
Done.